src/repositories/order/actions.ts

```
@@ -225,6 +226,31 @@ export class OrderUpdateHandler
 		resource.orderNumber = orderNumber;
 	}
 
+	setParcelCustomField(
+		context: RepositoryContext,
+		resource: Writable<Order>,
+		{ parcelId, name, value }: OrderSetParcelCustomFieldAction,
+	) {
+		assert(resource.shippingInfo, "shippingInfo is not defined");
```

I think we need to return a proper error here if this is required (same as CT)